### DISCUSSION TITLE
Display game summary in game detail

### DISCUSSION THREAD

**Discussion by oherau** [OnionUI/Onion#1528] [Ideas]
As of now when we browse in the game list of a system and we press D-Right, the cover and game title are display leaving an empty space for extra information.
It would be nice to have a summary of the game (or trivia or any other info related to the game) below the title in a textbox
Those data could be stored in a folder called **Infos** next to the covers folder (Imgs) and retreived by the screenscraper or any other way.

**Comment by ahmadtv** [comment]
Great idea !

**Comment by Nighted** [comment]
This was mentioned before. It can't be done because of the Miyoo source code being closed. You think Miyoo would make it open source considering that Onion sells most of its units.

**Reply by oherau** [reply]
I don't know exactly what is based on the core functions / cannot be changed and what can be tunned up / modified / overriden.
If the Miyoo is not open source there are a very few options left..

**Comment by k911a** [comment]
Who solved this problem? Is it possible to improve this section by adding text information about the game?
